liquidator/package.json

```diff
@@ -1,10 +1,29 @@
 {
-  "name": "liquidator",
-  "version": "0.1.0",
-  "description": "Liquidator",
+  "name": "@umaprotocol/liquidator",
+  "version": "1.0.0",
+  "description": "UMA Liquidator",
   "private": true,
   "dependencies": {
-    "winston": "^3.2.1",
-    "winston-transport": "^4.3.0"
+    "dotenv": "^6.2.0"
+  },
+  "devDependencies": {
+    "sinon": "^9.0.2",
+    "winston": "^3.2.1"
+  },
+  "homepage": "https://umaproject.org",
+  "license": "AGPL-3.0-or-later",
+  "main": "index.js",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.com/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/UMAprotocol/protocol.git"
+  },
+  "scripts": {
+    "test": "echo \"Error: run tests from root\" && exit 1"
```

Comment:
I wasn't sure whether to populate this `scripts::test` property but if so then this seems incorrect. It should be something like `truffle test/*`.

Reply:
Yeah, this is the default lerna blurb that tells you to run all tests from root. Are you suggesting that we allow users to run tests from root _and_ from each package? I think that would definitely make development easier since you aren't forced to run the entire test suite constantly.

Reply:
I think if we want these to be runnable from the subpackages, we should just remove the test command for now because truffle test won't work right now.

Reply:
Yeah wdyt? Intuitively it seems like a nice refactor to just run tests for each package

Reply:
Yeah, I think it would be nice. I just added a stand-in that says no tests are currently configured. We can add in the truffle bit once it works. Does that seem reasonable in the interim?